src/game/scenes/melee.c

```diff
@@ -308,7 +400,58 @@ void handle_action(scene *scene, int player, int action) {
                 }
             }
             break;
+        case ACT_KICK | ACT_UP:
+            if(local->page == PILOT_SELECT && check_pilot_stat_cheat(scene, player) &&
+               local->cheat_pilot_stats_stat[player] > 0) {
+                progressbar_set_highlight(local->bar_stat[player][local->cheat_pilot_stats_stat[player]], 0);
+                local->cheat_pilot_stats_stat[player]--;
+            }
+            break;
+        case ACT_KICK | ACT_DOWN:
+            if(local->page == PILOT_SELECT && check_pilot_stat_cheat(scene, player) &&
+               local->cheat_pilot_stats_stat[player] < 2) {
+                progressbar_set_highlight(local->bar_stat[player][local->cheat_pilot_stats_stat[player]], 0);
+                local->cheat_pilot_stats_stat[player]++;
+            }
+            break;
+        case ACT_KICK | ACT_LEFT:
+            if(local->page == PILOT_SELECT && check_pilot_stat_cheat(scene, player)) {
+                sd_pilot *pilot = game_state_get_player(scene->gs, player)->pilot;
+                int stat = local->cheat_pilot_stats_stat[player];
+                uint8_t *stat_val = get_pilot_stat(pilot, stat);
+                // reallocate points to other stats from selected stat
+                for(int other = 0; other < STAT_COUNT; other++) {
+                    uint8_t *other_val = get_pilot_stat(pilot, other);
+                    if(other == stat || *other_val >= MAX_STAT || *stat_val <= 0)
+                        continue;
+                    (*other_val)++;
+                    (*stat_val)--;
+                }
+                refresh_pilot_stats(scene, player);
+            }
+            break;
+        case ACT_KICK | ACT_RIGHT:
+            if(local->page == PILOT_SELECT && check_pilot_stat_cheat(scene, player)) {
+                sd_pilot *pilot = game_state_get_player(scene->gs, player)->pilot;
+                int stat = local->cheat_pilot_stats_stat[player];
+                uint8_t *stat_val = get_pilot_stat(pilot, stat);
+                // reallocate points from other stats into selected stat
+                for(int other = 0; other < STAT_COUNT; other++) {
+                    uint8_t *other_val = get_pilot_stat(pilot, other);
+                    if(other == stat || *other_val <= 0 || *stat_val >= MAX_STAT)
+                        continue;
+                    (*other_val)--;
+                    (*stat_val)++;
+                }
+                refresh_pilot_stats(scene, player);
+            }
+            break;
         case ACT_KICK:
+            if(local->page == PILOT_SELECT && check_pilot_stat_cheat(scene, player)) {
+                progressbar_set_highlight(local->bar_stat[player][local->cheat_pilot_stats_stat[player]], 1);
+                break;
+            }
+            // [[fallthrough]]
```

Comment:
GCC doesn't seem to like this

Reply:
CI passed. Do you mean to say, GCC doesn't like if you uncomment the `[[fallthrough]]`?
We're not setting the C standard to C23, so GCC should reject it.

Reply:
I was unable to compile this branch locally with GCC 13.2.0

Reply:
GCC 13 defaults to `-std=gnu23`, but we do explicitly set CMAKE_C_STANDARD to 11 in CMakeLists.
Could you post GCC's error message, and check that -std=c11 or -std=gnu11 is present?

Reply:
After consulting the manual and godbolt compiler explorer, I think this should now compile with your GCC 13.2.0

Reply:
Confirmed.